static/routes.config.php

```diff
@@ -184,9 +184,10 @@
 	],
 
 	'/notification'         => [
-		'[/]'            => [Module\Notifications\Notification::class, [R::GET]],
-		'/view/{id:\d+}' => [Module\Notifications\Notification::class, [R::GET]],
-		'/mark/all'      => [Module\Notifications\Notification::class, [R::GET]],
+		'[/]'               => [Module\Notifications\Notification::class, [R::GET]],
+		'/view/{id:\d+}'    => [Module\Notifications\Notification::class, [R::GET]],
```

Comment:
The `/view/{id:\d+}` and `/action/{id:\d+}` routes can be combined in a single `/(id:\d+}` route that accepts GET and POST.

Reply:
nope, because `/mark/all` would brick it .. The router couldn't decide if the parameter is either a variable or the "mark" sub-route.

Reply:
Even with the regex? Even by ordering the `/(id:\d+}` route after `/mark/all`? I'm a little disappointed.

Reply:
you got me .. I first used it without the `\d+`.. Now it works :-))) .. tested with some notifications on my local machine => no route failures :)